R/vars_manager.R

```diff
@@ -67,11 +77,37 @@ get_vector_var <- function(obj, name, abbrev_len=DEFAULT_ABBREV_LEN) {
     ))
 }
 
-get_matrix_var <- function(obj, name, abbrev_len=DEFAULT_ABBREV_LEN) {
+get_matrix_var <- function(obj, name, abbrev_len=DEFAULT_ABBREV_LEN, sort_by=NULL, ascending=NULL) {
+    # sort_by should be a vector of column indexes
+    # ascending should be a vector of booleans, or a single boolean
     dims <- dim(obj)
     summary <- sprintf("Size: %dx%d Memory: %s", dims[[1]], dims[[2]], human_bytes(utils::object.size(obj)))
     abbrev <- !is.null(abbrev_len) && length(obj) > abbrev_len
-    obj_pre <- `if`(abbrev, obj[1:ceiling(abbrev_len/dims[[2]]),], obj)
```

Comment:
Was this a bug? I couldn't tell if I was just misunderstanding something here, but it seemed like it wasn't getting the right values for pagination.
You'll see below (on lines 106 and 108) I changed this to be `obj[1:abbrev_len,]` instead of `obj[1:ceiling(abbrev_len/dims[[2]]),]`
So now, if you send in an `abbrev_len` of 5, and the matrix or dataframe has 10 rows, you'll get 5 of them back, and I verified that in the tests. Let me know if I'm missing something there

Reply:
you made it work like callisto-python now, so 👍